crates/bevy_pbr/src/render/shadow_sampling.wgsl

```diff
@@ -26,6 +26,40 @@ fn sample_shadow_map_hardware(light_local: vec2<f32>, depth: f32, array_index: i
 #endif
 }
 
+// Numbers determined by trial and error that gave nice results.
+const SPOT_SHADOW_TEXEL_SIZE: f32 = 0.0134277345;
```

Comment:
These aren’t texel sizes, they’re fudge factors right?

Reply:
In the case of spot shadows, which I didn't change, it was passed into `sample_shadow_map` as `texel_size`. In other invocations of `sample_shadow_map`, the actual texel size is passed into that parameter. So either it actually is the texel size or something is misnamed. What do you want me to do here?

Reply:
In the case of point light PCF I renamed it to `SCALE`.

Reply:
Ah, I see it now. Hmm, leave it as it is for now. We can fix it separately.